test/lib/jdk/test/lib/SA/SATestUtils.java

```diff
@@ -208,18 +205,12 @@ private static boolean canPtraceAttachLinux() throws IOException {
         // SELinux deny_ptrace:
         var deny_ptrace = Paths.get("/sys/fs/selinux/booleans/deny_ptrace");
         if (Files.exists(deny_ptrace)) {
-            try {
-                var bb = AccessController.doPrivileged(
-                    (PrivilegedExceptionAction<byte[]>) () -> Files.readAllBytes(deny_ptrace));
-                if (bb.length == 0) {
-                    throw new Error("deny_ptrace is empty");
-                }
-                if (bb[0] != '0') {
-                    return false;
-                }
-            } catch (PrivilegedActionException e) {
-                IOException t = (IOException) e.getException();
-                throw t;
+            var bb = Files.readAllBytes(deny_ptrace);
```

Comment:
Hello Roger, I'm guessing the `@SuppressWarnings("removal")` on this method will no longer be needed?